strings/strings.json

```diff
@@ -7614,5 +7614,13 @@
   "UpdatingResults": {
     "en": "Updating results",
     "fr": "Chargement des résultats"
+  },
+  "QuerySuggestionsAvailable": {
+    "en": "{0} search suggestion<pl>s are</pl> available.",
```

Comment:
Should we include the query when there is one?

Reply:
I don't think so.